packages/yoga/src/Snackbar/web/Snackbar.jsx

```diff
@@ -0,0 +1,190 @@
+
+import { Close } from '@gympass/yoga-icons';
+
+import { theme } from '../../Theme';
+
+import Box from '../../Box';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import Text from '../../Text';
+
+const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+
+  margin-right: ${theme.spacing.xxsmall}px;
+`;
+
+const IconButtonWrapper = styled.div`
+  display: flex;
+  align-items: center;
+
+  cursor: pointer;
+
+  &:hover {
+    svg {
+      fill: ${theme.colors.text.secondary};
+    }
+  }
+`;
+
+const ActionsWrapper = styled.aside`
+  display: flex;
+  align-items: center;
+
+  svg {
+    margin-left: ${theme.spacing.small}px;
+    margin-right: ${theme.spacing.zero};
+  }
+`;
+
+const StyledSnackbar = styled.div`
+  ${({
+    variant,
+    theme: {
+      yoga: {
+        components: { snackbar },
+      },
+    },
+  }) => `
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+  
+      position: absolute;
+      bottom: ${snackbar.position.mobile.bottom}px;
+      right: ${snackbar.position.mobile.right}px;
+      left: ${snackbar.position.mobile.left}px;
+  
+      min-width: ${snackbar.minWidth.mobile}px;
+      max-width: ${snackbar.maxWidth.default}px;
+  
+      min-height: ${snackbar.height.min}px;
+      max-height: ${snackbar.height.max}px;
+  
+      padding: ${snackbar.padding.default}px;
+      
+      border-radius: ${snackbar.border.radius}px;
+  
+      box-shadow: ${snackbar.shadow.default};
+
+      background-color: ${snackbar.variant.color[variant]};
+
+      z-index: 100;
+    `}
+
+  ${media.md`
+    left: auto;
+    bottom: ${theme.components.snackbar.position.desktop.bottom}px;
+    right: ${theme.components.snackbar.position.desktop.right}px;
+
+    width: ${theme.components.snackbar.minWidth.desktop}px;
+  `}
+`;
+
+const Snackbar = ({
+  open,
+  autoClose,
+  variant,
+  hideIcon,
+  message,
+  actionLabel,
+  onAction,
+  onClose,
+  theme: {
+    yoga: {
+      components: { snackbar },
+    },
+  },
+  ...props
+}) => {
+  useEffect(() => {
+    let timer;
+
+    if (open && autoClose && onClose) {
+      timer = setTimeout(() => {
+        onClose();
+      }, autoClose);
+    }
+
+    return () => clearTimeout(timer);
+  }, [open, autoClose]);
+
+  return (
+    open && (
+      <StyledSnackbar variant={variant} {...props}>
+        {!hideIcon && (
+          <IconWrapper>
+            <Icon
+              as={snackbar.variant.icon[variant]}
+              fill="secondary"
+              width="large"
+              height="large"
+            />
+          </IconWrapper>
+        )}
+
+        <Box as={Text.Small} flex={1} mr="small" numberOfLines={2}>
+          {message}
+        </Box>
+
+        <ActionsWrapper>
+          {onAction && actionLabel && (
+            <Button.Link onClick={onAction} secondary small>
+              {actionLabel}
+            </Button.Link>
+          )}
+
+          {onClose && (
```

Comment:
Is it possible to have the Snackbar with the autoClose feature but no showing the close button (the X icon)?

Reply:
Yes! I notice that it was missing and I have already adjusted, could you check again?

Reply:
Well done! :sparkles: